### DISCUSSION TITLE
Feed Translation to English #4095

### DISCUSSION THREAD

**Discussion by aligundogar** [FreshRSS/FreshRSS#6400] [Q&A] [answered]
I'm using Docker version of RSS in Linux server. With RSS Bridge and other web scrapers to RSS feed. But I want to add Russian, Arabic, Ukrainen feeds to my app with English translation of it. How can I do it?

My structure is,
Reverse Proxy, Cloud flare, RSS Bridge and other additional web scrapers to RSS.

**Comment by Alkarex** [comment]
This would be a job for a FreshRSS extension or for a Web browser feature / extension.

There is an example of translation to Chinese https://github.com/FreshRSS/Extensions/blob/master/README.md#by-jacob2826 , https://github.com/jacob2826/FreshRSS-TranslateTitlesCN

Automatic translation in your Web browser might also do the job (e.g. https://www.mozilla.org/en-US/firefox/features/translate/ ), especially if you put all articles of the same language under the same FreshRSS category.

**Reply by aligundogar** [reply]
I saw that but Im using Fresh RSS and Feed Me app. Browser extantion wouldnt work. Chinese ext is not workin with ENG I tested it before. Do you have anymore suggestion? Should I open issue for it?

**Reply by Alkarex** [reply]
You could adapt the extension to translate to English instead, or even better, propose a PR to chose target language

**Reply by math-GH** [reply, marked as answer]
I haven't tested it but maybe another service could to what you need: translating of an RSS feed and importing its translated feed into FreshRSS: https://rss.app/translate-rss-feeds

**Reply by math-GH** [reply]
Self-hosting RSS translation app: https://github.com/rss-translator/RSS-Translator (haven't tested it)
